content/updates/2020-12/3-Export-product-with-assets.md

```diff
@@ -0,0 +1,16 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Exports your products with their assets
+::: meta-data type="New" features="Collaboration, Productivity" available="late December" link-to-doc="../articles/TODO"
+
+Collaborating around product data is now more efficient as it's now possible to export products and their associated assets from within the product grid or from your product export profiles. It will save you time whenever you need to share product information with your internal and external teams.
```

Comment:
```suggestion
Collaboration around product data is now more efficient as it's now possible to export products and their associated assets from the product grid or your product export profiles. It will save you time whenever you need to share product information with your internal and external teams.
```